internal/config/config.go

```diff
@@ -47,81 +48,9 @@ const (
 	detachedNamespace    = "_detached"
 )
 
-type v1NamespaceSpecContent struct {
-	Name         string        `yaml:"name,omitempty"`
-	ControlPlane ControlPlane  `yaml:"controlPlane,omitempty"`
-	Agents       []Agent       `yaml:"agents,omitempty"`
-	Created      string        `yaml:"created,omitempty"`
-	Connectors   []interface{} `yaml:"connectors,omitempty"`
-}
-
-func updateConfigToK8sStyle() error {
```

Comment:
will re-add this functionality and add tests asap. Types have changed so this would no longer work as is